docs/source/tutorials/using_text.rst

```diff
@@ -0,0 +1,193 @@
+
+
+.. _using-text-objects:
+
+Rendering Text without using LaTeX 
+**********************************
+
+The simplest way to add text to your animations is to use the :class:`~.Text`
+class. It uses the `Pango library`_ to render text. With Pango, you can also
+render non-English alphabets like `你好` or  `こんにちは` or `안녕하세요` or
+`مرحبا بالعالم`.
+
+Here, is a simple Hello World animation.
+
+.. manim:: HelloWorld
+
+    class HelloWorld(Scene):
+        def construct(self):
+            text = Text('Hello world').scale(3)
+            self.play(FadeIn(text))
+
+You can also use :class:`~.MarkupText` where PangoMarkup (:ref:`pangomarkup`)
+can be used to render text. For example,
+
+.. manim:: SingleLineColour
+    :save_last_frame:
+
+    class SingleLineColour(Scene):
+        def construct(self):
+            text = MarkupText(
+               f'all in red <span fgcolor="{YELLOW}">except this</span>',
+               color=RED
+            )
+            self.add(text)
+
+.. _Pango library: https://pango.gnome.org
+
+Working with :class:`~.Text`
+==========================
+
+Text objects behave like a VGroup-like iterable of all characters in the given
+text. In particular, slicing is possible.
+
+For example, you can set each letter to different colour by iterating it.
+
+.. manim:: IterateColor
+
+    class IterateColor(Scene):
+        def construct(self):
+            text = Text("Colours").scale(2)
+            for letter in text:
+                letter.set_color(random_bright_color())
+                self.play(Write(letter))
+
+.. warning::
+
+    Please note that `Ligature`_ can cause problems here. If you need a
+    one-one mapping of characters to submobjects you should use 
+    ``disable_ligatures`` parameter in :class:`~.Text` while rendering.
+
+.. _Ligature: https://en.wikipedia.org/wiki/Ligature_(writing)
+
+Using Gradients
+---------------
+
+You can use Gradient using the :attr:`~.Text.gradient`. The value must
+be a Iterable or any Length.
+
+For example,
+
+.. manim:: GradientExample
+    :save_last_frame:
+
+    class GradientExample(Scene):
+        def construct(self):
+            t = Text("Hello", gradient=(RED, BLUE, GREEN)).scale(2)
+            self.add(t)
+
+
+.. _pangomarkup:
+
+What is PangoMarkup?
+====================
+
+PangoMarkup is a small markup language like html and it helps you avoid using
+"range of characters" while colouring or styling a piece a Text. You can use
+this language with :class:`~.MarkupText`.
+
+A simple example of a marked-up string might be::
+
+    <span foreground="blue" size="x-large">Blue text</span> is <i>cool</i>!"
+
+and it can be used with :class:`~.MarkupText` as 
+
+.. manim:: MarkupExample
+    :save_last_frame:
+
+    class MarkupExample(Scene):
+        def construct(self):
+            text = MarkupText('<span foreground="blue" size="x-large">Blue text</span> is <i>cool</i>!"')
+            self.add(text)
+
```

Comment:
I think everything from "MarkupElaborateExample" on is too detailed to be part of a tutorial.
I have in mind that tutorials should give you a broad overview of what manim is capable to do, and as `MarkupText` is one example class, it is appropriate to mention it in the tutorials, however, I would not explain all the possible flags and special use cases, that should go to the reference module.

Reply:
Sure, I will move all on ` What is PangoMarkup?` to MarkupText's reference and add simpler example's of using MarkupText text.